recipes/qvina/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+set -x
+
+mkdir build/
+cd build/
+
+cmake .. \
+ -DCMAKE_BUILD_TYPE=Release \
+ -DCMAKE_INSTALL_PREFIX=$PREFIX
+
+make -j $CPU_COUNT
+
```

Comment:
Is there a `make check`?

Reply:
Nope... what is it supposed to do? Testing?

Reply:
yeah usually